src/components/Button/Button.jsx

```diff
@@ -28,15 +28,15 @@ const Button = ({
   });
 
   const handleClick = e => {
-    onClick(e);
+    if (onClick) onClick(e);
```

Comment:
Just fixing an existing issue with Button. All 3 handlers aren't required, so checking for their presence before trying to fire.